src/freebsd/process.rs

```diff
@@ -129,6 +129,10 @@ impl ProcessExt for Process {
         self.cpu_usage
     }
 
+    fn total_accumulated_cpu_usage(&self) -> f32 {
+        FIXME
```

Comment:
For FreeBSd the CPU usage computation is a bit strange. I don't know if the `CPUSTATES` increase continuously or not. I'll let you check that. :)